src/test/java/edu/unc/lib/boxc/migration/cdm/SourceFilesCommandIT.java

```diff
@@ -160,7 +192,7 @@ public void generateUpdateAddSourceFileDryRunTest() throws Exception {
                 "-b", basePath.toString()};
         executeExpectSuccess(args2);
 
-        assertOutputMatches(".*New Files Mapped: +0.*");
+        assertOutputMatches(".*New Files Mapped: +1.*");
         assertOutputMatches(".*Total Files Mapped: +1.*");
```

Comment:
shouldn't the total files mapped be 2 since one got mapped in the first generate run, and then another one gets mapped in this dry run?